src/main/java/io/jenkins/plugins/audit/helpers/DateTimeHelper.java

```diff
@@ -0,0 +1,21 @@
+package io.jenkins.plugins.audit.helpers;
+
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.TimeZone;
+
+public class DateTimeHelper {
```

Comment:
Add `@Restricted(NoExternalUse.class)` to the class to mark this helper class module-private. We don't want to expose this API since it's internal details.